ui/packages/app/web/tailwind.config.js

```diff
@@ -1,8 +1,5 @@
 module.exports = {
-  purge: {
-    enabled: process.env.NODE_ENV === 'production',
```

Comment:
The `enabled` option is removed from the config schema as v3, as tailwind is not using `PurgeCSS` any more. 
Ref: https://tailwindcss.com/docs/upgrade-guide#configure-content-sources